RELEASING.md

```diff
@@ -95,19 +97,26 @@ Released as needed privately to individual vendors for critical security-related
 
 ## Binary Distributions
 
-### Windows
-* [ ] Download the artifacts from the [GitHub Actions "Test Windows" workflow](https://github.com/python-pillow/Pillow/actions/workflows/test-windows.yml)
-  and copy into `dist/`
-
-### Mac and Linux
+### macOS and Linux
 * [ ] Use the [Pillow Wheel Builder](https://github.com/python-pillow/pillow-wheels):
   ```bash
   git clone https://github.com/python-pillow/pillow-wheels
   cd pillow-wheels
   ./update-pillow-tag.sh [[release tag]]
   ```
 * [ ] Download wheels from the [Pillow Wheel Builder release](https://github.com/python-pillow/pillow-wheels/releases)
-  and copy into `dist/`
+  and copy into `dist/`. For example using [GitHub CLI](https://github.com/cli/cli):
+  ```bash
+  gh release download --dir dist --pattern "*.whl" --repo python-pillow/pillow-wheels
```

Comment:
I gather you were thinking that this command would be run from the main repository, so that all of the files would be in one neat 'dist' directory.

However, in the previous point, the user cloned and navigated into pillow-wheels. So I feel like this is confusing. It would be good to either clarify that this should be run from the main repository, or if this is run from pillow-wheels, then `--repo python-pillow/pillow-wheels` is unnecessary.

Reply:
That's right, I ran both these commands from the main repo, and agree this is confusing. Will clarify!

Reply:
Added https://github.com/python-pillow/Pillow/pull/7074/commits/cb68187006e02b3aeda44160338b67891455749e (and rebased in the PyQt workaround).